packages/ui-a11y-utils/src/KeyboardFocusRegion.ts

```diff
@@ -41,55 +41,49 @@ import { logError as error } from '@instructure/console'
 import keycode from 'keycode'
 
 import { scopeTab } from './scopeTab'
+import type { UIElement } from '@instructure/shared-types'
+import { FocusRegionOptions } from './FocusRegionOptions'
 
 class KeyboardFocusRegion {
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'element' implicitly has an 'any' type.
-  constructor(element, options) {
-    this._contextElement = findDOMNode(element)
-    // @ts-expect-error ts-migrate(2339) FIXME: Property '_options' does not exist on type 'Keyboa... Remove this comment to see the full error message
+  private readonly _options: FocusRegionOptions
+  private _focusLaterElement: Element | null = null
+  private _needToFocus = false
+  private _listeners: ReturnType<typeof addEventListener>[] = []
+  private _raf: RequestAnimationFrameType[] = []
+  private _active = false
+  private _wasDocumentClick?: boolean
+  public _contextElement?: Node
+
+  constructor(element: UIElement, options: FocusRegionOptions) {
+    this._contextElement = findDOMNode(element) as Node
```

Comment:
Can this somehow be `Window` in some cases?

Reply:
I think it can, but I dont see a use for using this on the whole `Window`. Later we could go over these casts and throw errors instead (now if you try to use it in a `Window` it will crash). But this will likely introduce breaking changes.

Reply:
Ok, thanks!